DigitalLearningSolutions.Web/Views/RegisterAtNewCentre/Confirmation.cshtml

```diff
@@ -31,12 +31,7 @@
       <partial name="_UnapprovedAccountWarning" />
     }
 
-    @if (Model.HasAdminAccountAtCentre) {
-      <p>
-        You will need to log out and in again before you can access the Learning Portal.
-      </p>
-      <partial name="_LogoutForm" model="@ButtonType.Secondary" />
-    } else if (Model.Approved && centreEmailIsVerified) {
+    @if (Model.HasAdminAccountAtCentre || Model.Approved && centreEmailIsVerified) {

```

Comment:
HasAdminAccountAtCentre => Approved so I'd say get rid of `Model.HasAdminAccountAtCentre ||`.

Reply:
But we still want to show admins the switch centre button if their email is unverified right? So it would be Model.HasAdminAccountAtCentre || (Model.Approved && centreEmailIsVerified)

Reply:
We'll show Switch Centre to all approved registrants.